### PR TITLE
Fix w textarea input

### PR DESCRIPTION
**Pull request by Nitsorn**
fixing w-textarea-input metadata to be of type Text and not basic Input, and update textarea font size to match twitchalertscss

### CODE REVIEW

**Comment by Nitsorn** [comment]
will do another PR after removing it from submodules

File: app/components/shared/widget-inputs/WTextAreaInput.vue.ts [hunk 1]
```diff
@@ -15,5 +16,9 @@ export default class WTextAreaInput extends WInput<string, IWTextMetadata> {
   @Prop({ default: () => ({}) })
   metadata: IWTextMetadata;
 
-
+  getValidations() {
+    return {
+      ...super.getValidations(),
+    };
+  }
```
Comment (holiber) [inline app/components/shared/widget-inputs/WTextAreaInput.vue.ts]: Hmm, do we really need that here?

Comment (Nitsorn) [reply app/components/shared/widget-inputs/WTextAreaInput.vue.ts]: true :) just copy pasted. removing now

Comment (holiber) [reply app/components/shared/widget-inputs/WTextAreaInput.vue.ts]: WTextAreaInput extends WInput that already has `getValidations` method

**Comment by andycreeth** [comment]
`yarn-error.log` was committed.  I've been meaning to add that to the `.gitignore` file.

**Comment by Nitsorn** [comment]
@andycreeth will add it, thanks

File: app/components/shared/widget-inputs/WTextAreaInput.vue [hunk 2]
```diff
@@ -1,11 +1,18 @@
 <template>
-  <textarea
+  <span>
+    <textarea
       type="text"
       :placeholder="metadata.placeholder"
```
Comment (holiber) [inline app/components/shared/widget-inputs/WTextAreaInput.vue]: Little fix that I forgot about - replace `metadata.placeholder` to `options.placeholder`. The `metadata` property can be undefined while `options` always exist.  After that, it's good for me.

Comment (Nitsorn) [reply app/components/shared/widget-inputs/WTextAreaInput.vue]: @holiber done! thanks for info

**Comment by andycreeth** [comment]
Hmm weird, `yarn-error.log` was added to the gitignore, but it's also still listed in the diff. I think you might need to delete it.

**Pull request merged**